### ISSUE TITLE
Diff between Revisions

### ISSUE THREAD

**Issue by sfrique** [databus23/helm-diff#6]
Hello,

It would be great if it was possible to diff between already released versions.

Something like this, current release history:
```
$ helm history winsome-owl                                                                                                                                             [11:04:28]
REVISION        UPDATED                         STATUS          CHART           DESCRIPTION     
1               Wed Apr 12 18:23:28 2017        SUPERSEDED      api-0.1.0    Install complete
2               Wed Apr 12 18:25:55 2017        SUPERSEDED      api-0.1.0    Upgrade complete
3               Wed Apr 12 18:30:48 2017        SUPERSEDED      api-0.1.0    Upgrade complete
4               Wed Apr 12 18:32:39 2017        SUPERSEDED      api-0.1.0    Rollback to 2   
5               Thu Apr 13 00:08:00 2017        DEPLOYED        api-0.1.0    Upgrade complete
```
So to get a diff between revision 5 (current and 3)

```
$ helm diff winsome-owl 3
```
Or explicit get the diff between 4 and 1
```
$ helm diff winsome-owl 4-1
```

Thanks

**Comment by aananthraj** [comment]
Waiting for this feature..!!
Mean while, as a work around , we could use the `helm get` and  `sdiff`/`vimdiff` to view the difference between two revisions side by side.

**Using sdiff**:
```bash
$ sdiff <(helm get $release --revision $revision_1 ) <(helm get $release --revision $revision_2)
 ```
To display only the changes use `-s` and `-w` flags
```bash
$ sdiff -s -w 80 <(helm get $release --revision $revision_1 ) <(helm get $release --revision $revision_2)
 ```

**Using vimdiff**:
```bash
$ vimdiff <(helm get $release --revision $revision_1 ) <(helm get $release --revision $revision_2)
 ```
Example:
```bash
$ sdiff <(helm get my_release --revision 4 ) <(helm get my_release --revision 5 )
REVISION: 4						      |	REVISION: 5
RELEASED: Fri Dec 15 08:59:48 2017			      |	RELEASED: Fri Dec 15 09:02:09 2017
...                                                           | ....
```

**Comment by aananthraj** [comment]
hi @sfrique,
I have implemented the above mentioned feature in this PR https://github.com/databus23/helm-diff/pull/21, have a look and let me know in case of corrections/bugs.

**Comment by sfrique** [comment]
@aananthraj  Thanks a lot to take sometime to implement this!

Unfortunately I don't have the time to test it now =/
But when i get a change i will.

I am currently working on some charts, I may try to use this to see if it will help!

Thanks again for the effort!

**Comment by aananthraj** [comment]
@sfrique any updates..?
your inputs would be helpful for closing this issue.

**Comment by sfrique** [comment]
I will test it at beginning of April.
Sorry about the long delay.

**Comment by sfrique** [comment]
@aananthraj this PR says it's messed up.
What should i do?

**Comment by aananthraj** [comment]
@sfrique , can you be brief..?

**Comment by sfrique** [comment]
@aananthraj i was checking PR #21 
Sorry, will test once i have some time!

**Comment by databus23** [comment]
I believe this has been fixed by a536dfc and is shipped since version 2.9.0+1 of this plugin.

**Comment by PhilipSchmid** [comment]
Small note to the comment from @codenio: You know have to use `helm get values` since `helm get` alone does not have a `--revision` flag (anymore?) (tested with Helm `v3.6.0`). Example:

```
vimdiff <(helm get values prometheus-operator --revision 14) <(helm get values prometheus-operator --revision 13)
```
